session.go

```diff
@@ -219,7 +219,8 @@ func (s *session) setup(
 		return nil, nil, err
 	}
 
-	s.packer = newPacketPacker(s.connectionID, s.cryptoSetup, s.connectionParameters, s.streamFramer, s.perspective, s.version)
+	s.packer = newPacketPacker(s.connectionID, s.cryptoSetup, s.connectionParameters, s.streamFramer,
+		s.perspective, s.version)
```

Comment:
"Don't worry about overflowing a punched card."

I strongly prefer to keep a consistent style across the project, which means either 1 line or 6 lines. I don't think C++ is a good reference point when talking about readability of code anyway.